Vagrantfile

```diff
@@ -0,0 +1,39 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+VAGRANTFILE_API_VERSION = '2'
+
+Vagrant.require_version '>= 1.5.0'
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+  config.vm.hostname = 'instance-image'
+  if Vagrant.has_plugin?('vagrant-omnibus')
+    config.omnibus.chef_version = 'latest'
+  end
+  if Vagrant.has_plugin?('vagrant-berkshelf')
+    config.berkshelf.enabled = true
+  end
+  config.vm.box = 'chef/centos-6.6'
```

Comment:
This is using the HashiCorp Atlas service, so it basically works like test kitchen. So no need to specify the URL. I probably should update the required Vagrant version to match the minor version I'm using since I tested it with that.